### DISCUSSION TITLE
How to auto rescale the Y-axis upon browser window resize

### DISCUSSION THREAD

**Discussion by somguyth** [reactivemarkets/react-financial-charts#547] [Q&A] [answered]
The x-axis (horizontal width) of the chart will auto rescale very nicely as I increase or decrease the size of the browser window.

Is there a way to achieve the same thing with the y-axis (vertical height) of the chart? The default behavior is that the height doesn't adjust at all when the browser window size is adjusted, which can lead to the bottom of the chart being chopped off. For example:

![image](https://user-images.githubusercontent.com/37128022/125687479-63ee90c7-9382-4022-a20c-aad0f3ce5f73.png)

**Comment by markmcdowell** [comment, marked as answer]
This will be somewhere in your code, if you check the examples they do resize.

e.g.
https://reactivemarkets.github.io/react-financial-charts/?path=/story/features-full-screen--daily
